eng/fv/tool-setup.rst

```diff
@@ -0,0 +1,315 @@
+
+This alias is used subsequently in this documentation.
+
+To check for a successful tool build, invoke the command without any 
+arguments, which should result in an extended help message being displayed:
+
+.. code-block:: shell
+
+  (env) prompt % tbuild
+  USAGE:
+  help - more details about usage and commands below
+  all modelname - runs clean, spin, gentests, copy, compile and run
+  clean modelname - remove spin, test files
+  archive modelname - archives spin, test files
+  zero  - remove all tesfiles from RTEMS
+  spin modelname - generate spin files
+  gentests modelname - generate test files
+  copy modelname - copy test files and configuration to RTEMS
+  compile - compiles RTEMS tests
+  run - runs RTEMS tests
+
+The tool is not yet ready for use, as it needs to be configured.
+
+Tool Configuration
+------------------
+
+Tool configuration involves setting up a new testsuite in RTEMS, and providing
+information to ``tbuild`` that tells it where to find key locations, and some
+command-line arguments for some of the tools.
+A template file ``testbuilder-template.yml`` is included,
+and contains the following entries:
+
+.. code-block:: python
+
+  # This should be specialised for your setup, as testbuilder.yml,
+  # located in the same directory as testbuilder.py
+  # All pathnames should be absolute
+
+  spin2test: <spin2test_directory>/spin2test.py
+  rtems: <path-to-main-rtems-directory>  # rtems.git, or ..../modules/rtems/
+  rsb: <rsb-build_directory>/rtems/6/bin/
+  simulator: <path-to>/sparc-rtems6-sis
+  testyamldir: <rtems>/spec/build/testsuites/validation/ # directory containing <modelname>.yml
+  testcode: <rtems>/testsuites/validation/
+  testexedir:  <rtems>/build/.../testsuites/validation/ # directory containing ts-<modelname>.exe
+  testsuite: model-0
+  simulatorargs: -leon3 -r s -m 2  # run command executes "<simulator> <simargs> <testexedir>/ts-<testsuite>.exe"
+  spinallscenarios: -DTEST_GEN -run -E -c0 -e # trail generation "spin <spinallscenarios> <model>.pml"
+
+This template should be copied/renamed to ``testbuilder.yml`` 
+and each entry updated as follows:
+
+* spin2test: 
+    This should be the absolute path to ``spin2test.py`` 
+    in the Promela sources directory.
+
+    ``/.../formal/promela/src/spin2test.py``
+
+
+
+* rtems: 
+    This should be the absolute path to your RTEMS source directory,
+    with the terminating ``/``.
+    From ``rtems-central`` this would be:
+
+    ``/.../rtems-central/modules/rtems/``
+
+    For a separate ``rtems`` installation 
+    it would be where ``rtems.git`` was cloned.
+
+    We refer to this path below as ``<rtems>``.
+
+* rsb:
+    This should be the absolute path 
+    to your RTEMS source-builder binaries directory,
+    with the terminating ``/``.
+    From ``rtems-central`` this would be (assuming RTEMS 6):
+
+    ``/.../rtems-central/modules/rsb/6/bin/`` 
+  
+* simulator: 
+    This should be the absolute path to the ``sis`` simulator:
```

Comment:
Does it have to be sis?

Why not use run scripts out of rtems-tools?

Reply:
No it doesn't - it's just what we used. I am not familiar with rtems-tools

For now I can just remove references to `sis`.